Stripe/STPPaymentContext.m

```diff
@@ -21,6 +21,19 @@
 #import "STPPaymentConfiguration+Private.h"
 
 #define FAUXPAS_IGNORED_IN_METHOD(...)
+#define FAUXPAS_IGNORED_IN_CLASS(...)
```

Comment:
My weak preference would be to move this into a different file, but feel free to ignore.